MCU_Microchip_SAMD.dcm

```diff
@@ -0,0 +1,81 @@
+EESchema-DOCLIB  Version 2.0
+#
+$CMP SAMD09C13A-SS
+D ARM Cortex-M0+ MCU, 48MHz, 8KB Flash, 4KB RAM, 2.4-3.6V, 12 GPIO, SOIC-14
+K ARM Cortex-M0+ SAMD09
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42414-SAM-D09_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD09D14A-M
+D ARM Cortex-M0+ MCU, 48MHz, 16KB Flash, 4KB RAM, 2.4-3.6V, 22 GPIO, QFN-24
+K ARM Cortex-M0+ SAMD09
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42414-SAM-D09_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10C13A-SS
+D ARM Cortex-M0+ MCU, 48MHz, 8KB Flash, 4KB RAM, 1.6-3.6V, 12 GPIO, SOIC-14
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10C14A-SS
+D ARM Cortex-M0+ MCU, 48MHz, 16KB Flash, 4KB RAM, 1.6-3.6V, 12 GPIO, SOIC-14
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10D13A-M
+D ARM Cortex-M0+ MCU, 48MHz, 8KB Flash, 4KB RAM, 1.6-3.6V, 22 GPIO, QFN-24
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10D13A-SS
+D ARM Cortex-M0+ MCU, 48MHz, 8KB Flash, 4KB RAM, 1.6-3.6V, 18 GPIO, SOIC-20
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10D14A-M
+D ARM Cortex-M0+ MCU, 48MHz, 16KB Flash, 4KB RAM, 1.6-3.6V, 22 GPIO, QFN-24
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10D14A-SS
+D ARM Cortex-M0+ MCU, 48MHz, 16KB Flash, 4KB RAM, 1.6-3.6V, 18 GPIO, SOIC-20
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD10D14A-U
+D ARM Cortex-M0+ MCU, 48MHz, 16KB Flash, 4KB RAM, 1.6-3.6V, 18 GPIO, WLCSP-20
+K ARM Cortex-M0+ SAMD10
+F http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42242-SAM-D10_Datasheet.pdf
+$ENDCMP
+#
+$CMP SAMD11C14A-SS
+D ARM Cortex-M0+ MCU, 48MHz, 16KB Flash, 4KB RAM, 1.6-3.6V, 18 GPIO, SOIC-20
```

Comment:
Small hiccup: this a SOIC-14 package, not SOIC-20.

Reply:
thank you for pointing that out, fixed.